doc/release-notes-namecoin.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 0.21
 
+- `name_show` now (by default) shows an error for expired names. This can be
+  overridden by setting the allowExpired RPC option to true, or by using the
```

Comment:
Sorry, another nit:  Please put `` around `allowExpired` here as well (as with the command-line argument later).